### DISCUSSION TITLE
firmware update creality CR 6SE

### DISCUSSION THREAD

**Discussion by Turfite** [community/community#40988] [Other Feature Feedback, Questions, & Ideas]
Hi guys hope you are all well, I am wondering if I can install newest firmware or do I have to install one update after another, for example if I have firmware 3 can I jump straight to update 6 or do I have to install update 4 & 5

Thank you in advance

Turfite

**Comment by Akash1134** [comment]
Hi @Turfite , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by Turfite** [reply]
Thank you Akash I have a Creality CR-6 SE 3d printer the version is V2.0.1.8     please could you tell me or give me the links to upgrade this firmware version, I have looked into this but maybe me, but the dates on internet seem to be old. what it is is the bed leveling, I have tried factory reset, to no avail, the z axis seems to rise on on side when self leveling, I have sorted screws etc, 

Many thanks in advance,

Turfite

**Comment by Turfite** [comment]
Many thanks Akash

Kind regards
